src/main/java/com/tailf/jnc/YangUnion.java

```diff
@@ -147,7 +147,7 @@ public boolean canEqual(Object obj) {
      */
     @Override
     public boolean equals(Object obj) {
-        return canEqual(obj)
+        return obj instanceof YangUnion
```

Comment:
Ditto - if `canEqual` changes, should this code too?

Reply:
Right, the change felt a bit unclean, but primitive enough to "excuse" imho.
I will roll-back this, and if we need to silence the lint later ill ad lint skip...